examples/module1/lesson5/tanstack-query/components/ArticleAuthors.tsx

```diff
@@ -0,0 +1,32 @@
+import { useQuery } from '@tanstack/react-query';
+
+function ArticleAuthors() {
+  const { isPending, error, data } = useQuery({
```

Comment:
od razu takie query do osobnego hooka, może kiedyś będziesz chciał jego użyć ponownie
jest to jakaś logika, która nie powinna interesować tego komponentu i nie ma co go zaśmiecać